source-map.bs

```diff
@@ -433,21 +427,35 @@ However, It is unclear what a "source map reference" looks like in anything othe
 More specifically, what a source map reference looks like in a language that doesn't support
 JavaScript-style single-line comments.
 
-JSON over HTTP Transport
-========================
-
-For historic reasons, when delivering source maps over HTTP, servers may prepend a line
-starting with the string `)]}'` to the source map.  A client fetching a source map via
-HTTP thus should check if the response starts with this string and then ignore the
-first line.
-
-```
-)]}'garbage here
-{"version": 3, ...}
-```
-
-Is to be interpreted as
-
-```
-{"version": 3, ...}
-```
+Fetching Source Maps {#fetching-source-maps}
+============================================
+
+To fetch a source map given a [=URL=] |url|, run the following steps:
+
+1. Let |promise| be [=a new promise=].
+1. Let |request| be a new [=request=] whose [=request/URL=] is |url|.
+1. [=Fetch=] |request| with [=processResponseConsumeBody=] set to the following steps given [=response=] <var ignore>response</var> and null, failure, or a [=byte sequence=] |bodyBytes|:
+    1. If |bodyBytes| is null or failure, [=reject=] |promise| with a {{TypeError}} and abort these steps.
+    1. If |url|'s [=url/scheme=] is an [=HTTP(S) scheme=] and |bodyBytes| [=byte sequence/starts with=] \`<code>)]}'</code>\`, then:
+        1. [=While=] |bodyBytes|'s [=byte sequence/length=] is not 0 and |bodyBytes|'s 0th byte is not an [=HTTP newline byte=]:
+            1. remove the 0th byte from |bodyBytes|.
+
+            <div class="note">
+            <span class="marker">Note:</span> For historic reasons, when delivering source maps over HTTP(S), servers may prepend a line
```

Comment:
Nope, this is not a note, this is normative text :) The fact that it's historic might be a note, but the parsing instructions are not a note.

Reply:
Right: there are the parsing instructions in the lines above, and this note explains what's the reason for them.

Reply:
oh! sorry, I can't read.